papis/web/epubjs.py

```diff
@@ -0,0 +1,67 @@
+import os.path
+
+import dominate.tags as t
+
+import papis.web.html as wh
+import papis.web.static
+
+VIEWER_PATH = "epubjs-reader/reader/index.html"
+EPUBJS_URL = "https://github.com/futurepress/epubjs-reader"
+
+
+def detect_local_installation() -> bool:
+    for path in papis.web.static.static_paths():
+        viewer = os.path.join(path, VIEWER_PATH)
+        if os.path.exists(viewer):
+            return True
+    return False
+
+
+def error_message() -> str:
+    """Error message for when there is no epubjs-reader installation"""
+    return f"""
+No installation of epubjs found.
+
+If you want to be able to read and see EPUB files from within
+the papis web applications you need to install epubjs-reader.
+
+You can download the EPUBJS bundle from the url
+
+    {EPUBJS_URL}
+
+and extract it to your config directory under 'web/epubjs-reader', for instance
+in '~/.config/papis/web/epubjs-reader/'.
+
+On linux and mac you can simply run the following lines
+
+    mkdir -p ~/.config/papis/web/
+    git clone {EPUBJS_URL} ~/.config/papis/web/epubjs-reader
+    """
+
+
+def widget(_unquoted_file_path: str) -> None:
```

Comment:
```suggestion
def widget(unquoted_file_path: str) -> None:
```
? Why underscore it?